style.css

```diff
@@ -23,13 +27,68 @@ ul {
 	list-style: none;
 }
 
-a {
+a:not(.item-link) {
 	display: block;
 	text-align: center;
-	color: #a6a2c5;
-	text-transform: uppercase;
+	color: var(--color);
+	padding: 15px;
+	text-decoration: none;
+	font-size: 25px;
+}
+
+.item-link {
 	padding: 15px;
 	text-decoration: none;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.item-icon {
+	margin-bottom: 5px;
+	transform: translateY(0px);
+	transition-duration: 450ms;
+}
+
+.item-text {
 	font-size: 25px;
+	color: var(--color);
+	font-family: sans-serif;
+	position: absolute;
+	opacity: 0;
+	transform:translateY(0) ;
+	transition-duration: 500ms;
 }
 
+.item-dot {
+	display: block;
+	width: 8px;
+	height: 8px;
+	border-radius: 50%;
+	background-color: var(--color);
+	opacity: 0;
+	transform:scale(0);
+	transition:all 500ms;
+}
+
+.item-link.active .item-icon {
+	/* Select the element with the className "item-link" and "active" and "item-icon"?*/
```

Comment:
No, Select the `item-icon` when the `.item-link` has the `.active` class. I highly recommend you to learn and practice more about CSS specificity :) 

This is a nice way to learn while playing a game. https://flukeout.github.io/

Reply:
Coud we also use this synthax ?

 .item-link:active + .item-icon {
}

Reply:
Or something like this ?
.item-link:active, .item-icon {
}

Reply:
Yes that syntax is also valid but it means different type of selections.

Reply:
Thanks :).
I was reading about css specificity and I was wondering If the above one could work.

Reply:
What do you mean by "it means different type of selections"?

Reply:
- `.a .b` is [descedant selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Descendant_combinator)
- `.a + .b` is [adjacent sibling combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator) 
- `.a:active` is [a pseudo class](https://developer.mozilla.org/en-US/docs/Web/CSS/:active)

More [CSS selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors)

Reply:
Thanks. I added the link to my favorites!